win/Projects/libsndfile/config.h

```diff
@@ -23,61 +23,10 @@ typedef unsigned __int64  uint64_t;
 #define ssize_t           SSIZE_T
 #define __func__          __FUNCTION__
 
```

Comment:
It seemed related since the changes in this pull request were meant to align the code with the supported Visual Studio versions.   Since the _MSC_VER is tested against 1500 exactly, the code only applies when building with VS2008's compiler.  Since the functions in question were added to MS' math.h in VS2013 anyway, there didn't seem much point in keeping it around.